mobile/src/main/java/org/openhab/habdroid/ui/WriteTagActivity.java

```diff
@@ -48,14 +51,21 @@
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
+import java.util.Locale;
 
 public class WriteTagActivity extends AppCompatActivity {
     private static final String TAG = WriteTagActivity.class.getSimpleName();
+    public static final String QUERY_PARAMETER_ITEM_NAME = "i";
+    public static final String QUERY_PARAMETER_STATE = "s";
+    public static final String QUERY_PARAMETER_MAPPED_STATE = "m";
```

Comment:
What's the semantic difference of those two? Looks like the same value is passed for both.